src/components/common/Modal/ModalFolderItem.tsx

```diff
@@ -37,7 +37,17 @@ const CheckIcon = styled.img`
   right: 8px;
 `;
 
-function ModalFolderItem({ folder, selected, onClick }) {
+interface ModalFolderItemProps {
+  folder: any;
```

Comment:
r: 타입스크립트에서 any 사용을 지양해주세요. any는 모든 타입을 포함하는 최상위 타입이라 타입스크립트의 타입 체크 기능을 안쓰는 것과 동일합니다. unknown으로 지정후 todo 주석을 달아 추후 타입을 꼭 정의해주세요.

```suggestion
  /**
   * @todo Folder 타입 정의
   */
  folder: unknown;
```